marklogic-client-api/src/test/java/com/marklogic/client/test/LineSplitterTest.java

```diff
@@ -0,0 +1,100 @@
+package com.marklogic.client.test;
+
+import com.marklogic.client.DatabaseClient;
+import com.marklogic.client.DatabaseClientFactory;
+import com.marklogic.client.datamovement.LineSplitter;
+import com.marklogic.client.io.Format;
+import com.marklogic.client.io.StringHandle;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.InputStreamReader;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.stream.Stream;
+import java.util.zip.GZIPInputStream;
+
+import static org.junit.Assert.*;
+
+public class LineSplitterTest {
+    static final private String jsonlFile = "src/test/resources/data" + File.separator + "line-delimited.jsonl";
+    static final private String jsonlGzipFile = "src/test/resources/data" + File.separator + "line-delimited.jsonl.gz";
+    static final private String xmlFile = "src/test/resources/data" + File.separator + "line-delimited.txt";
+    private DatabaseClient client;
+
+    @Before
+    public void setUp() {
+        client = DatabaseClientFactory.newClient("localhost", 8012,
+                new DatabaseClientFactory.DigestAuthContext("rest-admin", "x"));
```

Comment:
If the test uses com.marklogic.client.test.Common.connect() to get a database client, we can maintain the authentication in one place.

Unless there's another reason to use rest-admin, writing and reading should work with the less privileged rest-writer.

Reply:
I just realized that I don't need a connection to the database. I only checked the StringHandles returned by the splitter. I'm removing the connection part.